packages/squiggle-lang/__tests__/Distributions/Scale_test.res

```diff
@@ -0,0 +1,38 @@
+open Jest
+open Expect
+open TestHelpers
+
+describe("Scale logarithm", () => {
+  /* These tests may not be important, because scalelog isn't normalized
+  The first one may be failing for a number of reasons.
+ */
+  Skip.test("mean of the base e scalar logarithm of an exponential(10)", () => {
+    let rate = 10.0
+    let scalelog = DistributionOperation.Constructors.scaleLogarithm(
+      ~env,
+      mkExponential(rate),
+      MagicNumbers.Math.e,
+    )
+
+    let meanResult = E.R2.bind(DistributionOperation.Constructors.mean(~env), scalelog)
+    // expected value of log of exponential distribution.
+    let meanAnalytical = Js.Math.log(rate) +. 1.0
+    switch meanResult {
+    | Ok(meanValue) => meanValue->expect->toBeCloseTo(meanAnalytical)
+    | Error(err) => err->expect->toBe(DistributionTypes.OperationError(DivisionByZeroError))
+    }
+  })
+  let low = 10.0
+  let high = 100.0
+  let scalelog = DistributionOperation.Constructors.scaleLogarithm(~env, mkUniform(low, high), 2.0)
+
+  test("mean of the base 2 scalar logarithm of a uniform(10, 100)", () => {
```

Comment:
I'm confused; I'd expect the mean of the scale of a uniform to just be equal to the mean of the uniform

Reply:
This is also a weird case, as I try it here: 
<img width="1048" alt="image" src="https://user-images.githubusercontent.com/377065/167316837-5fa85547-3c08-4623-9ea7-03d499ddfeb2.png">